requirements.txt

```
@@ -6,3 +6,4 @@ autograd
 toml
 appdirs
 semantic_version==2.6
+dask[delayed]
```

```suggestion
dask[delayed]==2.9.2
```
Worth continuing the version pinning convention?

---

Good idea!